src/file/adapter.ts

```diff
@@ -82,6 +82,7 @@ export function rawFileMetadataToFileMetadata(data: RawFileMetadata): FileMetada
     accessTime: data.accessTime,
     modificationTime: data.modificationTime,
     blocksReference: base64toReference(data.fileInodeReference),
```

Comment:
even if the PR addressed this problem, the tag id is not part of the file nor directory metadata.